cryptography/src/bls12381/primitives/ops.rs

```diff
@@ -678,6 +712,31 @@ where
     V::verify(public, &hm_sum, signature)
 }
 
+// no_std version of aggregate_verify_multiple_messages which ignores the concurrency parameter and
+// runs in current thread.
+#[cfg(not(feature = "std"))]
+pub fn aggregate_verify_multiple_messages<'a, V, I>(
```

Comment:
Prefer you take the same approach with this function as `recover_signatures`, I think we can unify under a single instance of the function signature?